### PR TITLE
admin auth added with new controllers

### LINKED ISSUE #1961: Need to update controller and db as discussed in meeting

### PR DESCRIPTION
**Pull request by Ansh101112**
## Fixes Issue
Close #1961

As discuss in the meeting the specifications of the controllers to give access to the new db collection of admin has been added.

## Summary by CodeRabbit

- **New Features**
	- Enhanced admin login process with database lookup for improved security.
	- Expanded Admin model to include additional fields for better user management and security.

- **Bug Fixes**
	- Updated the login URL endpoint for development purposes to ensure proper connectivity during testing.

- **Style**
	- Minor formatting adjustments in the main application file for improved code readability.

### CODE REVIEW

File: src/User/pages/UserAuth/UserAuth.jsx
Comment (codervivek5) [inline src/User/pages/UserAuth/UserAuth.jsx:1]: it properly works in this localhost url then the code will merge ?

**Comment by Ansh101112** [comment]
> 
> 
> ## Fixes Issue
> #1960
> As discuss in the meeting the specifications of the controllers to give access to the new db collection of admin has been added.
> 
> 
> 
> 
> 
> ## Changes proposed
> 
> 
> 
> ## Screenshots
> 
> 
> 
> ## Note to reviewers
> 
> 
> 
> 
> 
> 
> ## Summary by CodeRabbit
> 
> - **New Features**
> 	- Enhanced admin login process with database lookup for improved security.
> 	- Expanded Admin model to include additional fields for better user management and security.
> 
> - **Bug Fixes**
> 	- Updated the login URL endpoint for development purposes to ensure proper connectivity during testing.
> 
> - **Style**
> 	- Minor formatting adjustments in the main application file for improved code readability.
> 
>

File: src/User/pages/UserAuth/UserAuth.jsx
Comment (Ansh101112) [reply src/User/pages/UserAuth/UserAuth.jsx:1]: Sir please tell elaborately

Comment (codervivek5) [reply src/User/pages/UserAuth/UserAuth.jsx:1]: "http://localhost:8080/api/auth/login",

ye address dekhna 
jo phle "https://vigybag-backend.onrender.com/ point kr rha tha to......local address to ni chalega n after merge

**Review by codervivek5 — CHANGES_REQUESTED** [review]

**Comment by Ansh101112** [comment]
@codervivek5 sir conflicts fixed

**Review by codervivek5 — APPROVED** [review]

**Pull request merged**